docs/user-docs/annotation.md

```diff
@@ -615,6 +622,7 @@ Supported JSON _option_ payload patterns:
 	- The _pathsuffix_ MAY join additional tables to the path and MAY project from these tables as well as the table bound to the `S` table alias.
 	- The _pathsuffix_ SHOULD reset the path context to `$S` if it has joined other tables.
 - `"selector_ux_mode"`: The display mode for the recordedit input field when this table is part of a foreignkey relationship as the `outbound` table. Supported values are `"facet-search-popup"` and `"simple-search-dropdown"`, with `"facet-search-popup"` being the default. Currently only supported in `entry` contexts.
+- `"bulk_create_foreign_key_candidates"`: Use this property to control the bulk selection of foreign key values in `entry/create` context when there is a prefill query parameter. Supported value is an array of foreign key `names` in the format of `[['schema_name', 'foreign_key_name'], ...]`. This will override the `bulk_create_foreign_key` property defined in the display annotation. Currently only supported in `entry/create` context.
```

Comment:
Doesn't this property also support `false` and `null`?

Reply:
The `bulk_create_foreign_key_candidates` property of `table-display` annotation is only an array of constraint names. The `bulk_create_foreign_key` property of `display` annotation can be defined on the table and allows for only `false` or `null` so I don't think the `*_candidates` property should do the same thing on the same model element.